### ISSUE TITLE
The instrument "Sousaphone in B♭" is misleading and not useful

### ISSUE THREAD

**Issue by lowtoves** [musescore/MuseScore#23226]
### Your idea

There is an instrument called "Sousaphone in B♭" which is in bass clef and transposed up by an octave and a major 2nd. It should be renamed to "Sousaphone (Transposing Bass Clef)".

### Problem to be solved

Sousaphonists who read transposed bass clef are part of an extreme minority. The most normal looking sousaphone instrument name in the list when you're making a new score is a transposing bass clef instrument. This trips up anyone new to Musescore who isn't familiar with playing sousaphone because when they bring that music to a sousaphonist, there's like a 99% chance it won't make any sense to the musician.

### Additional context

Nearly all sousaphone players play an instrument pitched in B♭ and read concert pitch bass clef. Not transposed by an octave or a major 2nd. British brass band musicians, some American marching band musicians, and trumpet players who are learning the sousaphone but do no wish to learn bass clef, read treble clef transposed up by two octaves and a major 2nd. This accounts for "Sousaphone (concert pitch)" and "Sousaphone (treble clef) in B♭"
According to https://github.com/musescore/MuseScore/pull/2569, some European brass bands read a transposed bass clef part, which became "Sousaphone in B♭". I cannot find any published or unofficial music written this way. I cannot find anyone in the mixed-culture Seattle / PNW street band scene who is familiar with sousaphone parts written in this way. The only reason I'm bothering filing an issue for this is because the bands I'm in use Musescore and keep accidentally using the instrument and becoming disheartened when no one can read it.

**Comment by Dean110** [comment]
### Additional context
In the tradition of European brass bands the lowest voice of the band is the B♭ bass, which is scored in a transposed treble clef like the B♭ sousaphone.  [This article describes the voices of the traditional brass band](https://www.bandsman.co.uk/writing.htm), and [this template shows an example of a blank brass band score](https://www.justmusicuk.com/images/Standard%20British%20Brass%20Band%20Instrumentation%20-%20Full%20Score.pdf) with the bottom voice being the B♭ bass.
### My un-researched assumptions:
I'm assuming as sousaphones became more popular in American marching bands and around the world, due to their more convenient shape for playing whilst standing, the brass bands of Europe started using them instead of tubas in bands and just kept the scoring the same.  So what you end up with at some point is an option for B♭ sousaphone as an option in scoring software instead of B♭ bass.  The purpose of the transposition in the scores of brass bands was so that a brass player could pick up a different voiced instrument that might be tuned up a fourth and still keep the original fingerings, a _C_ would be an open fingering regardless if the horn was tuned to F, B♭, or E♭.  This allowed for players to not need to transpose music from concert pitch to different fingerings when picking up a different horn to cover an open chair in the band.  Of the two lower voices of the brass band, the B♭ and E♭ basses, there are [E♭ sousaphones available](https://us.wessex-tubas.com/products/eb-sousaphone-4-valve-sp28), so it might be possible that you would see a both sousaphones on one score.  
### My suggestion:
Maybe instead of "Sousaphone (treble clef) in B♭", use "B♭ Bass"  if you think a change is still needed for most of the users here.

**Comment by lowtoves** [comment]
> ### Additional context
> In the tradition of European brass bands the lowest voice of the band is the B♭ bass, which is scored in a transposed treble clef like the B♭ sousaphone. [This article describes the voices of the traditional brass band](https://www.bandsman.co.uk/writing.htm), and [this template shows an example of a blank brass band score](https://www.justmusicuk.com/images/Standard%20British%20Brass%20Band%20Instrumentation%20-%20Full%20Score.pdf) with the bottom voice being the B♭ bass.
> 
> ### My un-researched assumptions:
> I'm assuming as sousaphones became more popular in American marching bands and around the world, due to their more convenient shape for playing whilst standing, the brass bands of Europe started using them instead of tubas in bands and just kept the scoring the same. So what you end up with at some point is an option for B♭ sousaphone as an option in scoring software instead of B♭ bass. The purpose of the transposition in the scores of brass bands was so that a brass player could pick up a different voiced instrument that might be tuned up a fourth and still keep the original fingerings, a _C_ would be an open fingering regardless if the horn was tuned to F, B♭, or E♭. This allowed for players to not need to transpose music from concert pitch to different fingerings when picking up a different horn to cover an open chair in the band. Of the two lower voices of the brass band, the B♭ and E♭ basses, there are [E♭ sousaphones available](https://us.wessex-tubas.com/products/eb-sousaphone-4-valve-sp28), so it might be possible that you would see a both sousaphones on one score.
> 
> ### My suggestion:
> Maybe instead of "Sousaphone (treble clef) in B♭", use "B♭ Bass" if you think a change is still needed for most of the users here.

I agree that "B♭ Bass" would be a good name for the "Sousaphone (treble clef) in B♭" instrument, though also see value in how informative the current name is, so I don't particularly mind. However my request isn't about that, it's about the instrument "Sousaphone in B♭", which uses transposed bass clef and is not the tradition anywhere as far as I can find, other than in the bands that the person who added the instrument played in. I would love to learn about this practice from anyone with experience, though in the meantime I still believe the name should be changed for all the reasons I outlined.

**Comment by henkdegroot** [comment]
Your statement "Nearly all sousaphone players play an instrument pitched in B♭ and read concert pitch bass clef." is not correct.
This really depends on the part of the world/country you life. In the Netherlands (I know a really small country) sousaphone Bb and tuba Bb are read in pitched notation Mostly using a bass clef. To my knowledge the treble clef is only used for UK Brassband settings, in which almost every instrument is written in treble clef and pitched notation. And it is very rare to have a Sousaphone in  a UK Brassband. There is always a Bb Tuba and an Eb Tuba both reading treble and instrument pitch.

So why do you want the Sousaphone in Bb renamed?

There is already a Sousaphone (concert pitch) version, which you really should use. Which in the instrument dialog has an incorrect description, as it indicates it is noted in treble clef but when you use it, it actually is using bass clef.

The solution to make this work for everyone could be just to have an indicator in the dialog if you want to have the selected instrument in pitched notation or concert pitch.

**Comment by henkdegroot** [comment]
In the Netherlands there a number of different band types. The brassband (originated from the UK) is always reading transposed treble. The other band type in the Netherland, wind band, concert band, etc. all uses transposed bass clef. It wasn't until I started using MuseScore that I found we are a minority in doing this. Not sure how this started, I can only guess because in the Netherlands we might believe reading music is hard enough without manually transpising while reading. One benefit I believe there is, is that the valves to press are then the same regardless of the sheetmusic. When I play an Eb Sousaphone or a Bb Sousaphone, when there is a G written in the staff it is played without pressing any. So easier to switch from one instrument to another.

I beleive I read somewhere that they want to change the way the instrument selection works, not sure how and it that will resolve this.  I do agree it could help to be more clear in the dialog but then again, how would the person know what to select?

There is no need to changes the Sousaphone in Bb to Bb Bass, are really a Sousaphone is not a Bass and the Tuba Bb already exists as well.
Perhaps Bb Sousaphone (concert pitcch) would help some people to select this version, instead of select Sousaphone in Bb. 

Thinking about this, perhaps this can also be resolved by adding a Sousapone in Eb (for treble and bass clef). As with that you get the dropdown selection for the transposing key. 

I found this for your information: https://www.vsl.info/en/academy/brass/bass-tuba#notation
And this article as well: https://music.stackexchange.com/questions/102039/transposing-bass-clef-instruments-for-tuba-bombardon-and-contrabass

**Comment by shoogle** [comment]
> instrument called "Sousaphone in B♭" [...] should be renamed to "Sousaphone **(Transposing Bass Clef)**"

Within the New Score dialog, the word "in" before a pitch means "transposing in" that pitch.

We could rename it to "Sousaphone (bass clef) in B♭" so that users don't assume it's the default Sousaphone.

> instead of "Sousaphone **(treble clef)** in B♭", use "B♭ Bass"

"B♭ Bass" might be a suitable name to use in the score, but it's not descriptive enough to be used in the New Score dialog, and it needs to begin with 'S' to sort near the other sousaphones.

**Comment by azmeuk** [comment]
> According to https://github.com/musescore/MuseScore/pull/2569, some European brass bands read a transposed bass clef part, which became "Sousaphone in B♭".

I am the author of #2569 and I can give a little context here.
I have been a sousaphone players for more than ten years in the south-west of France, and I can assure that the extreme majority of sousaphone scores I met in my different bands are in B♭ with a bass clef transposition of one octave. This is not just me, all the other sousaphonist I met too. This is the case for [Bandas](https://fr.wikipedia.org/wiki/Banda_(musique)) which are popular local bands in the south-west of France, for [student fanfares](https://fr.wikipedia.org/wiki/Fanfare_des_Beaux-Arts), which are student brass bands across all the country, and for different other local brass band traditions.
The only scores I have seen in C were from classical musicians occasionally playing street music, but they were in bass clef too.

> I cannot find any published or unofficial music written this way.

I am not sure you will find many. Generally those bands keeps their scores, because it a marker of the identity of the bands, rarely shared except for very popular songs.

**Comment by SteveBlower** [comment]
Similar to this issue https://github.com/musescore/MuseScore/issues/22877
